<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Marc Noguera-Julian, PhD. TreeTopUnder" />

<meta name="date" content="2022-04-30" />

<title>Análisis de la subpoblación de osos</title>

<script src="site_libs/header-attrs-2.13/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">UAB_LLops</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="01_index.html">Introducción</a>
</li>
<li>
  <a href="02_Exploratory_v1.html">Analisis Exploratorio</a>
</li>
<li>
  <a href="03_Osos_v1.html">Población Osos</a>
</li>
<li>
  <a href="04_about.html">Conclusiones</a>
</li>
<li>
  <a href="MetadataExplorer.html">Metadatos</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Análisis de la subpoblación de osos</h1>
<h4 class="author">Marc Noguera-Julian, PhD. TreeTopUnder</h4>
<h4 class="date">2022-04-30</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-04-30
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>UAB_LLops/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version 1.7.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown is ignored by Git. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220310code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20220310)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220310code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220310)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomTreeTopUnderUABLLopstree9ca78cd48e8732fe2db42779e73314e0160eca85targetblank9ca78cda"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/TreeTopUnder/UAB_LLops/tree/9ca78cd48e8732fe2db42779e73314e0160eca85" target="_blank">9ca78cd</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomTreeTopUnderUABLLopstree9ca78cd48e8732fe2db42779e73314e0160eca85targetblank9ca78cda" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/TreeTopUnder/UAB_LLops/tree/9ca78cd48e8732fe2db42779e73314e0160eca85" target="_blank">9ca78cd</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/01_index.Rmd
    Ignored:    analysis/02_Exploratory_v1.Rmd
    Ignored:    analysis/03_Osos_v1.Rmd
    Ignored:    analysis/04_about.Rmd
    Ignored:    analysis/Osos_vs_lobos_v1.Rmd
    Ignored:    analysis/_book/
    Ignored:    analysis/_main.Rmd
    Ignored:    analysis/figure/
    Ignored:    code/.DS_Store
    Ignored:    code/100_Report_v1.2.Rmd
    Ignored:    code/myFunctions.R
    Ignored:    data/.DS_Store
    Ignored:    data/processed/
    Ignored:    data/ps_silva.rds
    Ignored:    data/ps_silva_RR.rds
    Ignored:    data/raw/
    Ignored:    output/.DS_Store
    Ignored:    output/Deliverables/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/03_Osos_v1.Rmd</code>) and HTML (<code>docs/03_Osos_v1.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/TreeTopUnder/UAB_LLops/9ca78cd48e8732fe2db42779e73314e0160eca85/docs/03_Osos_v1.html" target="_blank">9ca78cd</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-30
</td>
<td>
Final commit
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/TreeTopUnder/UAB_LLops/7913ffe507709039e7a7dea658fcc93715a7b246/docs/03_Osos_v1.html" target="_blank">7913ffe</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-24
</td>
<td>
final version
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>Como objetivo principal del estudio procedemos a analizar las muestras fecales obtenidas de ejemplares de osos. En el conjunto de muestras encontramos varias muestras obtenidas de un mismo individuo asi como invididuos de los que se ha obtenido una sola muestra. Asi pués,aracterizaremos la beta-diversidad apareada entre muestra de un mismo individuo y la compararemos con la beta-diversidad de individuos diferentes En el conjunto de individuos muestreados, usaremos las variables <em>Edad</em> (Osezno, Subadulto y Adulto) y <em>Sexo</em> (Macho y Hembra) para detectar diferencias significativas a nivel de alfa-diversidad o de abundancia relativa en generos bacterianos.</p>
<div id="beta-diversidad" class="section level2">
<h2>Beta-Diversidad</h2>
<div id="ordenacion" class="section level3">
<h3>Ordenacion</h3>
<p>Usando el mismo método de anàlisis que en la sección previa, usamos la similitud entre la composición del microbioma fecal para realizar un análisi de ordenacion.</p>
<p><em>Deliverables: NMDS_StressPlot_osos.pdf, Ordination_OsoSamples.pdf</em></p>
<p>No se observa una agrupación ni por grupos de edad, region ni por sexo en el grupo de muestras de osos.</p>
<p><img src="figure/03_Osos_v1.Rmd/ordenacion%20osos-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-ordenacion-osos-1">
Past versions of "ordenacion osos-1.png"
</button>
</p>
<div id="fig-no-spaces-ordenacion-osos-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/TreeTopUnder/UAB_LLops/blob/7913ffe507709039e7a7dea658fcc93715a7b246/docs/figure/03_Osos_v1.Rmd/ordenacion osos-1.png" target="_blank">7913ffe</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="permanova" class="section level3">
<h3>PERMANOVA</h3>
<p><em>Deliverables: Adonis_Oso.xls</em></p>
<p>Para testear estadísticamente la influencia de las variables secundarias (Edad, Sexo) en la estructura composicional del microbioma fecal usamos el test de ANOVA permutacional (vegan::adonis) sobre las muestras de Oso.</p>
<p>No se detecta una asociación entre el grupo de edad, el sexo o la region geografica y la composición del microbioma (p&gt;0.05). Tampoco se detecta relación con el individuo asociado a la muestra, después de eliminar del dataset los individuos con una sola muestra.</p>
<pre><code>[1] TRUE</code></pre>
<table>
<thead>
<tr class="header">
<th align="left"></th>
<th align="left">term</th>
<th align="right">df</th>
<th align="right">SumOfSqs</th>
<th align="right">R2</th>
<th align="right">statistic</th>
<th align="right">p.value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Sexo.1</td>
<td align="left">Sexo</td>
<td align="right">1</td>
<td align="right">0.1942048</td>
<td align="right">0.0208859</td>
<td align="right">1.5785218</td>
<td align="right">0.134</td>
</tr>
<tr class="even">
<td align="left">Edad.1</td>
<td align="left">Edad</td>
<td align="right">2</td>
<td align="right">0.1652095</td>
<td align="right">0.0178969</td>
<td align="right">0.6560285</td>
<td align="right">0.788</td>
</tr>
<tr class="odd">
<td align="left">Region.1</td>
<td align="left">Region</td>
<td align="right">2</td>
<td align="right">0.3497258</td>
<td align="right">0.0376114</td>
<td align="right">1.4264690</td>
<td align="right">0.147</td>
</tr>
<tr class="even">
<td align="left">Individuo.1</td>
<td align="left">Individuo</td>
<td align="right">18</td>
<td align="right">2.0470145</td>
<td align="right">0.3321253</td>
<td align="right">0.9393196</td>
<td align="right">0.611</td>
</tr>
</tbody>
</table>
</div>
<div id="intra-vs-inter" class="section level3">
<h3>Intra vs Inter</h3>
<p>En este análisis pretendemos detectar si la variabilidad en la composición del microbioma entre especímenes del mismo individuo es significativamente diferente que la composición entre especímenes de diferentes individuos en la población de osos de la que se han recogido muestras. Con este objetivo, calculamos las distancias WUnifrac para cada par de muestras, y a cada valor de distancia lo etiquetamos como proveniente del mismo individuo (Intra) o de diferentes individuos (Inter) para despues comparar la distribución de estos valors y sus diferencias entre los grupos descritos.</p>
<p><em>Deliverables: BetaDiversity_intra.vs.inter_osos.pdf</em></p>
<p>No se observan diferencias estadísticamente significativas entre la variabilidad asociada a especimenes del mismo animal y especimenes de distintos animales en la población de osos. Asi pues la composición del microbioma intestinal de un mismo animal no es más ni menos estable que el de distintos animales. Este resultado es concordante con el resultado de la anova permutacional en relacion a la variable individuo y se puede explicar por la distancia temporal en el muestreo de un mismo animal o por efectos externos que pudieran modificar la composicion de las muestras.</p>
<p><img src="figure/03_Osos_v1.Rmd/beta-diversidad%20intra%20vs%20inter-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-beta-diversidad-intra-vs-inter-1">
Past versions of "beta-diversidad intra vs inter-1.png"
</button>
</p>
<div id="fig-no-spaces-beta-diversidad-intra-vs-inter-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/TreeTopUnder/UAB_LLops/blob/9ca78cd48e8732fe2db42779e73314e0160eca85/docs/figure/03_Osos_v1.Rmd/beta-diversidad intra vs inter-1.png" target="_blank">9ca78cd</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-30
</td>
</tr>
<tr>
<td>
<a href="https://github.com/TreeTopUnder/UAB_LLops/blob/7913ffe507709039e7a7dea658fcc93715a7b246/docs/figure/03_Osos_v1.Rmd/beta-diversidad intra vs inter-1.png" target="_blank">7913ffe</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="madre-hijos" class="section level3">
<h3>Madre-Hijos</h3>
<p>En este análisis se pretendre detectar si la similitud del microbioma intestinal entre animales que forman parte de un mismo nucleo familiar (el formado por una osa hembra y los oseznos a su cuidado) es mas elevada que la similitud entre individuos no conectados. Para definir que las dos muestras comparadas provienen de un “nucleo familiar”, requerimos:</p>
<ul>
<li>Las dos muestras son de individuos diferentes</li>
<li>Los dos individuos tienen una relación de parentesco Madre-hijo</li>
<li>Una de los individuos esta clasificado con Edad=“Osezno” en el momento de toma de muestra</li>
<li>Las dos muestras se han tomado en el mismo año (variable Year).</li>
</ul>
<p><em>Deliverables: BetaDiversity_madre_hijo_osos.pdf</em></p>
<p>No se observan diferencias en cuanto a la beta-diversidad comparando muestras de un mismo núcleo familiar o muestras de individuos independientes.</p>
<p><img src="figure/03_Osos_v1.Rmd/madre_hijo-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-madre_hijo-1">
Past versions of madre_hijo-1.png
</button>
</p>
<div id="fig-madre_hijo-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/TreeTopUnder/UAB_LLops/blob/9ca78cd48e8732fe2db42779e73314e0160eca85/docs/figure/03_Osos_v1.Rmd/madre_hijo-1.png" target="_blank">9ca78cd</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="análisis-de-las-variables-secundarias-edad-y-sexo." class="section level2">
<h2>Análisis de las variables secundarias (Edad y Sexo).</h2>
<p>En este análisis se responderá a las preguntas relacionadas con el cambio de la composición del microbioma (parametrizado con alfa diversidad o composición global) en relación a la edad de los individuos o al sexo de los mismos. Este análisis se realizara unicamente en la población de osos.</p>
<div id="comparativa-en-alfa-diversidad-entre-grupos-de-edad-y-sexo" class="section level3">
<h3>Comparativa en alfa-diversidad entre grupos de edad y sexo</h3>
<p>Comparativa no paramétrica (Wilcoxon/Kruskal-Wallis) en diversidad alfa (Chao1,Shannon) entre grupos de edad, sexo y Region</p>
<p><em>Deliverables: AlphaDiversity_osos.pdf</em></p>
<p>No se observan diferencias en cuanto a alfa-diversidad en los diferentes de grupos de edad en la población de osos muestreada. No se observan diferencias en cuanto a alfa-diversidad entre machos y hembras en la población de osos muestreada. No se observan diferencias en cuanto a alfa-diversidad entre regiones geográficas en la población de osos muestreada.</p>
<p><img src="figure/03_Osos_v1.Rmd/alfa-diversity%20vs%20age-1.png" width="480" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-alfa-diversity-vs-age-1">
Past versions of "alfa-diversity vs age-1.png"
</button>
</p>
<div id="fig-no-spaces-alfa-diversity-vs-age-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/TreeTopUnder/UAB_LLops/blob/9ca78cd48e8732fe2db42779e73314e0160eca85/docs/figure/03_Osos_v1.Rmd/alfa-diversity vs age-1.png" target="_blank">9ca78cd</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-30
</td>
</tr>
<tr>
<td>
<a href="https://github.com/TreeTopUnder/UAB_LLops/blob/7913ffe507709039e7a7dea658fcc93715a7b246/docs/figure/03_Osos_v1.Rmd/alfa-diversity vs age-1.png" target="_blank">7913ffe</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="abundancia-diferencial-entre-grupos-de-edad-y-sexo." class="section level3">
<h3>Abundancia diferencial entre grupos de edad y sexo.</h3>
<p>Comparativa (Wilcoxon) entre la abundancia de difernetes generos bacterianos entre grupos de edad y sexo y la valoración del <em>effect size</em> en la diferencia mediante el paquete <em>lefser</em>.</p>
<p><em>Deliverables: Lefse_vs_Edad.pdf, Lefse_vs_Sexo.pdf y boxplots independientes para cada género positivo al test</em></p>
<p>Se observan taxas diferencialmente abundantes entre grupos de edad (Adulto vs otros, Subadultos vs otros, Osezno vs Otros) mediante <em>lefser</em>(<a href="http://www.bioconductor.org/packages/release/bioc/html/lefser.html" class="uri">http://www.bioconductor.org/packages/release/bioc/html/lefser.html</a>).La abundancia relativa diferencial detectada, se da en estos casos en generos poco abundantes y se debe a un subgrupo reducido de individuos donde estos géneros se pueden detectar.</p>
<p>Se observan taxas diferencialmente abundantes entre machos y hembras en la población de osos muestreada, mediante <em>lefser</em>(<a href="http://www.bioconductor.org/packages/release/bioc/html/lefser.html" class="uri">http://www.bioconductor.org/packages/release/bioc/html/lefser.html</a>). La abundancia relativa diferencial detectada, se da en estos casos en generos poco abundantes y se debe a un subgrupo reducido de individuos donde estos géneros se pueden detectar.</p>
<p><img src="figure/03_Osos_v1.Rmd/DA%20vs%20Age%20Group%20and%20Sex-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-DA-vs-Age-Group-and-Sex-1">
Past versions of "DA vs Age Group and Sex-1.png"
</button>
</p>
<div id="fig-no-spaces-DA-vs-Age-Group-and-Sex-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/TreeTopUnder/UAB_LLops/blob/7913ffe507709039e7a7dea658fcc93715a7b246/docs/figure/03_Osos_v1.Rmd/DA vs Age Group and Sex-1.png" target="_blank">7913ffe</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="abundancia-diferencial-entre-especies-osos-vs-lobos" class="section level3">
<h3>Abundancia diferencial entre Especies (Osos vs Lobos)</h3>
<p>En este análisis adicional se pretende detectar los generos bacterianos que son diferencialmente abundantes entre Osos y Lobos, mediante el paquete <em>lefser</em>.</p>
<p><em>Deliverables: Lefse_vs_Especie.pdf yboxplots independientes para cada género positivo al test</em></p>
<p>Se observan taxas diferencialmente abundantes entre lobos y osos mediante <em>lefser</em>(<a href="http://www.bioconductor.org/packages/release/bioc/html/lefser.html" class="uri">http://www.bioconductor.org/packages/release/bioc/html/lefser.html</a>). En particular, generos como Escherichia-Shigella y Streptococcus son muy abundantes en Osos mientras que su presencia en muestras de lobos es infrecuente. El cambio en la abundancia relativa de estos generos es, pués, estructural y consistente.</p>
<p><img src="figure/03_Osos_v1.Rmd/bonus-track%20DA%20vs%20Especie-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-bonus-track-DA-vs-Especie-1">
Past versions of "bonus-track DA vs Especie-1.png"
</button>
</p>
<div id="fig-no-spaces-bonus-track-DA-vs-Especie-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/TreeTopUnder/UAB_LLops/blob/7913ffe507709039e7a7dea658fcc93715a7b246/docs/figure/03_Osos_v1.Rmd/bonus-track DA vs Especie-1.png" target="_blank">7913ffe</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Como ejemplo de géneros diferencialmente abundantes, se muestran las abundancias de dos de los géneros que mas diferencias presentan entre osos y lobos*. En el caso de los géneros <em>Escherichia-shigella</em> y <em>Streptococcus</em> se detecta un incremento consistente en el numero de <em>reads</em> asignados los dos géneros bacterianos(p&lt;0.001 para los dos generos). <img src="figure/03_Osos_v1.Rmd/include%20graphics%20escherichia-shigella%20and%20streptococcus-1.png" width="480" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-include-graphics-escherichia-shigella-and-streptococcus-1">
Past versions of "include graphics escherichia-shigella and streptococcus-1.png"
</button>
</p>
<div id="fig-no-spaces-include-graphics-escherichia-shigella-and-streptococcus-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/TreeTopUnder/UAB_LLops/blob/7913ffe507709039e7a7dea658fcc93715a7b246/docs/figure/03_Osos_v1.Rmd/include graphics escherichia-shigella and streptococcus-1.png" target="_blank">7913ffe</a>
</td>
<td>
Marc Noguera-Julian
</td>
<td>
2022-04-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.1.1 (2021-08-10)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] lefser_1.2.0                SummarizedExperiment_1.22.0
 [3] Biobase_2.54.0              GenomicRanges_1.44.0       
 [5] GenomeInfoDb_1.30.1         IRanges_2.28.0             
 [7] S4Vectors_0.32.3            BiocGenerics_0.40.0        
 [9] MatrixGenerics_1.4.3        matrixStats_0.61.0         
[11] xlsx_0.6.5                  patchwork_1.1.1            
[13] ggforce_0.3.3               ggplot2_3.3.5              
[15] dplyr_1.0.8                 phyloseq_1.38.0            

loaded via a namespace (and not attached):
  [1] backports_1.4.1        workflowr_1.7.0        plyr_1.8.6            
  [4] igraph_1.2.11          splines_4.1.1          gmp_0.6-4             
  [7] TH.data_1.1-0          kSamples_1.2-9         digest_0.6.29         
 [10] SuppDists_1.1-9.7      foreach_1.5.2          htmltools_0.5.2       
 [13] fansi_1.0.3            magrittr_2.0.3         memoise_2.0.1         
 [16] paletteer_1.4.0        cluster_2.1.2          Biostrings_2.62.0     
 [19] sandwich_3.0-1         colorspace_2.0-3       ggrepel_0.9.1         
 [22] xfun_0.30              libcoin_1.0-9          crayon_1.5.1          
 [25] RCurl_1.98-1.6         prismatic_1.1.0        jsonlite_1.8.0        
 [28] zeallot_0.1.0          zoo_1.8-10             survival_3.3-1        
 [31] iterators_1.0.14       ape_5.6-2              glue_1.6.2            
 [34] polyclip_1.10-0        gtable_0.3.0           zlibbioc_1.40.0       
 [37] XVector_0.34.0         DelayedArray_0.18.0    statsExpressions_1.3.0
 [40] Rhdf5lib_1.16.0        Rmpfr_0.8-7            scales_1.2.0          
 [43] mvtnorm_1.1-3          DBI_1.1.2              PMCMRplus_1.9.3       
 [46] Rcpp_1.0.8.3           performance_0.8.0      datawizard_0.3.0      
 [49] RColorBrewer_1.1-3     modeltools_0.2-23      ellipsis_0.3.2        
 [52] pkgconfig_2.0.3        reshape_0.8.8          rJava_1.0-6           
 [55] farver_2.1.0           multcompView_0.1-8     sass_0.4.1            
 [58] utf8_1.2.2             here_1.0.1             tidyselect_1.1.2      
 [61] labeling_0.4.2         rlang_1.0.2            reshape2_1.4.4        
 [64] later_1.3.0            effectsize_0.6.0.1     munsell_0.5.0         
 [67] tools_4.1.1            cachem_1.0.6           cli_3.2.0             
 [70] generics_0.1.2         ade4_1.7-18            broom_0.8.0           
 [73] evaluate_0.15          biomformat_1.22.0      stringr_1.4.0         
 [76] fastmap_1.1.0          BWStest_0.2.2          yaml_2.3.5            
 [79] rematch2_2.1.2         knitr_1.37             fs_1.5.2              
 [82] purrr_0.3.4            coin_1.4-2             WRS2_1.1-3            
 [85] nlme_3.1-155           whisker_0.4            correlation_0.8.0     
 [88] compiler_4.1.1         rstudioapi_0.13        tibble_3.1.6          
 [91] tweenr_1.0.2           bslib_0.3.1            stringi_1.7.6         
 [94] highr_0.9              parameters_0.17.0      forcats_0.5.1         
 [97] lattice_0.20-45        Matrix_1.4-0           vegan_2.5-7           
[100] permute_0.9-7          multtest_2.50.0        vctrs_0.4.1           
[103] pillar_1.7.0           lifecycle_1.0.1        rhdf5filters_1.6.0    
[106] mc2d_0.1-21            jquerylib_0.1.4        data.table_1.14.2     
[109] bitops_1.0-7           insight_0.16.0         httpuv_1.6.5          
[112] R6_2.5.1               promises_1.2.0.1       codetools_0.2-18      
[115] boot_1.3-28            MASS_7.3-55            assertthat_0.2.1      
[118] rhdf5_2.38.0           xlsxjars_0.6.1         rprojroot_2.0.2       
[121] withr_2.5.0            multcomp_1.4-18        GenomeInfoDbData_1.2.7
[124] mgcv_1.8-39            bayestestR_0.11.5      parallel_4.1.1        
[127] grid_4.1.1             tidyr_1.2.0            rmarkdown_2.13        
[130] git2r_0.30.1           ggstatsplot_0.9.1     </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
